opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -57,21 +55,31 @@ pub struct ZipkinPipelineBuilder {
 
 impl Default for ZipkinPipelineBuilder {
     fn default() -> Self {
+        let timeout = env::get_timeout();
         ZipkinPipelineBuilder {
             #[cfg(feature = "reqwest-blocking-client")]
-            client: Some(Box::new(reqwest::blocking::Client::new())),
+            client: Some(Box::new(
+                reqwest::blocking::Client::builder()
+                    .timeout(timeout)
+                    .build()
+                    .unwrap(),
```

Comment:
Maybe we can use `unwrap_or` to create a client without timeout if for some reason the build failed. It would be better to break ppl's application.

Reply:
Swapped for unwrap_or_else as unwrap_or is typically meant for an existing value. Otherwise shows a clippy warning.